### PR TITLE
fix(tests): fix invariants

### PR DESCRIPTION
**Pull request by PierrickGT**

### CODE REVIEW

**Comment by PierrickGT** [comment]
We need to exclude invariant tests from the coverage because the run would be extremely slow on the latest versions of forge.
See https://github.com/foundry-rs/foundry/issues/4952 for more info.

**Review by deluca-mike — APPROVED** [review]

File: test/invariant/stores/TimestampStore.sol [hunk 1]
```diff
@@ -11,7 +11,7 @@ contract TimestampStore {
         currentTimestamp = uint32(block.timestamp);
     }
 
-    function increaseCurrentTimestamp(uint32 timeJump_) external {
-        currentTimestamp += timeJump_;
+    function increaseCurrentTimestamp(uint32 timeJump_) external returns (uint32) {
+        return currentTimestamp += timeJump_;
```
Comment (toninorair) [inline test/invariant/stores/TimestampStore.sol:15]: looks suspicious, do we ever increase `currentTimestamp` in this store? 
If no, is this contract needed?

How timestamp is stored between multiple calls?

Comment (PierrickGT) [reply test/invariant/stores/TimestampStore.sol:15]: Yes we do and we use the stored value in the `useCurrentTimestamp` modifier:
https://github.com/MZero-Labs/protocol/blob/ad9ab6c53f5e0807c8df4a303e50476f69e6b91c/test/invariant/Invariant.t.sol?plain=1#L36

And also when checking the rate confidence interval:
https://github.com/MZero-Labs/protocol/blob/ad9ab6c53f5e0807c8df4a303e50476f69e6b91c/test/invariant/handlers/ProtocolHandler.sol?plain=1#L48

Comment (toninorair) [reply test/invariant/stores/TimestampStore.sol:15]: I saw it as `return currentTimestamp + timeJump_`

File: test/invariant/handlers/ProtocolHandler.sol [hunk 2]
```diff
@@ -48,26 +48,21 @@ contract ProtocolHandler is CommonBase, StdCheats, StdUtils, TestUtils {
...
         );
 
-        // Warping past the confidence interval knowingly can lead to overprinting (i.e. broken invariants).
-        timeJump_ = _bound(timeJump_, 0, _min(timeUntilRateConfidenceExpires_, 10 days));
-
-        _timestampStore.increaseCurrentTimestamp(uint32(timeJump_));
-        vm.warp(_timestampStore.currentTimestamp());
-
-        console2.log("--> time jump %s to %s", timeJump_, _timestampStore.currentTimestamp());
+        if (timeUntilRateConfidenceExpires_ == 0) return;
```
Comment (toninorair) [inline test/invariant/handlers/ProtocolHandler.sol:56]: Consider 
``` 
if (timeUntilRateConfidenceExpires_ == 0)  {
            _minterGateway.updateIndex();
            return;
 }
```

**Pull request merged**